R/PrettyOutput.R

```diff
@@ -91,7 +83,18 @@ unnumberAdditionalCriteria <- function(markdown) {
   return(markdown)
 }
 
-#' Does the given string of digit(s) indicate value larger than 40.
+#' Convert a string of an integer < 40 to a string of a corresponding roman numeral.
+as.roman <- function(digit_str) {
```

Comment:
@aki-nishimura -- why aren't you using `utils::as.roman()`?

Reply:
Gah, it somehow did not even occur to me that there is a built-in function for the purpose... thank you for pointing it out @msuchard. Now fixed.